icons/update.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="none"><g fill="#000" clip-path="url(#a)"><path d="M2.81 13.05a7.09 7.09 0 0 0 5.05 2.09h.15l-.03-1.5a5.55 5.55 0 0 1-4.11-1.65c-2.05-2.05-2.16-5.29-.39-7.5v2.84h1.5V2.34H0v1.5h2.06c-2 2.79-1.75 6.71.75 9.21m10.11-10.1A7.09 7.09 0 0 0 7.87.86v1.5c1.51 0 2.93.59 3.99 1.65 1.95 1.95 2.16 4.98.66 7.18V8.68h-1.5v4.98H16v-1.5h-2.33c2-2.79 1.75-6.7-.75-9.21"/></g><defs><clipPath id="a"><path fill="#fff" d="M0 0h16v16H0z"/></clipPath></defs></svg>
```

Comment:
Sorry I have missed this earlier:

> "Viewbox attribute must be present to allow scaling."

After the last _svgo_ pass the `viewBox` is not present anymore in any of the icons. Just mentioning it if it was important for downstream consumer use before this gets released.

Reply:
🤦‍♀️ https://github.com/mozilla/protocol-assets/issues/113